### ISSUE TITLE
Intellisense for project.json / global.json

### ISSUE THREAD

**Issue by nosami** [OmniSharp/omnisharp-roslyn#134]
We should be able to generate intellisense for these files based on json schemas and nuget feed information.

**Comment by nosami** [comment]
Just had a look at how the [Kulture](https://github.com/OmniSharp/Kulture/blob/master/Completions.py) plugin  works.

It's using hardcoded schema information. We should probably use the [project.json schema information](http://json.schemastore.org/project)

It's also using the following url for nuget information.
https://www.myget.org/F/aspnetrelease/api/v2/Packages()?$select=Id&$format=json&orderby=DownloadCount&$top=100

We probably also need to use https://nuget.org/api/v2/ and parse Nuget.Config for other feeds?

**Comment by nosami** [comment]
v3 feed are working well - http://api-v3search-0.nuget.org/query?q=omni

**Comment by masaeedu** [comment]
@nosami Is anyone assigned to this?

**Comment by nosami** [comment]
No. I did hear on the grapevine though that MS might contribute some code
to do this //cc @davidfowl

**Comment by david-driscoll** [comment]
We have the three new endpoints now `/packagesearch`, `/packageversion`, `/packagesource`.  Atom uses a fallback repo https://github.com/OmniSharp/omnisharp-nuget/tree/resources .

Should we have `packagesearch` use similar behavior to atom (using the fallback) or should that detail be left to each editor to decide?

If we can leave it alone, then we should be able to close this, otherwise we should make an enhancement to `packagesearch`
